README.md

```diff
@@ -2,6 +2,9 @@
 
 An all in one plugin for converting text case in Neovim. It converts a piece of text to an indicated string case and also is capable of bulk replacing texts without changing cases
 
+![CI/CD](https://github.com/johmsalas/text-case.nvim/actions/workflows/ci.yml/badge.svg)
```

Comment:
What about adding the versions here that we are testing against? (just as text)

Reply:
What a good idea!!
I'd rather read the version from the yml, but that's not possible using markdown. I think we'd have to duplicate the values in the workflow and the readme 🤔

Reply:
Yeah I also think that we have to duplicate.